### ISSUE TITLE
Cannot write multi-line methods

### ISSUE THREAD

**Issue by v0dro** [minad/iruby#52]
I tried to write multi-line methods in iruby similar to `irb` or `pry`, but this was the result:

```

In [42]: def hello a

In [43]: def hello a

In [44]: def hello a

```

it would be great to be able to do that!

**Comment by minad** [comment]
You mean in the iruby console? Do you know how irb/pry handles these incomplete expressions?

**Comment by v0dro** [comment]
Not really.

If you could just point me in the right direction I'll read it up and fix it.

**Comment by minad** [comment]
Closed in favor of SciRuby/iruby#5
